dashboards-observability/.cypress/integration/3_panels.spec.js

```diff
@@ -37,6 +37,39 @@ const moveToTestPanel = () => {
   cy.get('h1').contains(TEST_PANEL).should('exist');
   cy.wait(delay);
 };
+let i = 1;
```

Comment:
I still think, this variable needs renaming. If this takes care of count of panels, then name it accordingly.